components/x-timeline-feed/src/TimelineFeed.scss

```diff
@@ -0,0 +1,51 @@
+@import 'o-colors/main';
+@import 'o-grid/main';
+@import 'o-typography/main';
+
+.articleGroup {
+	border-top: 4px solid #000;
+
+	@include oGridRespondTo($from: M) {
+		display: grid;
+		grid-gap: 0 20px;
+		grid-template: "heading articles" min-content / 1fr 3fr;
+	}
+}
+
+.articleGroup__heading {
+	@include oTypographySansBold($scale: 1);
+	@include oTypographyMargin($top: 2, $bottom: 2);
+
+	@include oGridRespondTo($from: M) {
+		grid-area: heading;
+	}
+}
+
+.articleGroup__articles {
+	list-style-type: none;
+	padding: 0;
+	@include oTypographyMargin($top: 2);
+
+	@include oGridRespondTo($from: M) {
+		grid-area: articles;
+	}
+}
+
+.article {
+	display: flex;
+	justify-content: space-between;
+	border-bottom: 1px solid oColorsGetPaletteColor('black-20');
+	@include oTypographyMargin($bottom: 2);
+
+	:global {
+		.o-teaser {
+			border-bottom: 0;
+			@include oTypographyPadding($bottom: 0);
+		}
```

Comment:
If possible I think we should try to avoid components "reaching into" other components. This sort of thing lacks visibility and will never get cleaned up.

Reply:
I agree, though I couldn't see a way around it in this case.

Reply:
have this override as a separate class that your component passes to `x-teaser`?

Reply:
Ah, I didn't realise x-teaser supported this. I'll do that. Thanks